pkg/kube/inject/inject.go

```diff
@@ -399,8 +400,6 @@ func RunTemplate(params InjectionParameters) (mergedPod *corev1.Pod, templatePod
 		return nil, nil, err
 	}
 
-	proxyUID, proxyGID := GetProxyIDs(params.namespace)
```

Comment:
this call should not be removed, we want this behavior in cluster-wide deployments. To make it work in multitenant mode, we should wrap it in an if-clause that uses the other method (derive UID/GID from pod, `GetProxyIDsFromPod()`)